datacite-update-doi-handler/src/main/java/no/unit/nva/datacite/handlers/UpdateDoiEventHandler.java

```diff
@@ -113,9 +115,8 @@ private void handleDoi(DoiStateDto doiStateDto, DoiUpdateRequestEvent input, Doi
                            PublicationApiClientException e) {
```

Comment:
can we rename this also so that it says what it handles, like handleGoneDoi or something